ci-operator/step-registry/openshift-qe/installer/azure/ipi/ovn/ipsec/provision/openshift-qe-installer-azure-ipi-ovn-ipsec-provision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: openshift-qe-installer-azure-ipi-ovn-ipsec-provision
```

Comment:
Similar to the deprovision I think  we can put all these steps directly into the pre steps in the main workflow

Reply:
Done